### PR TITLE
Version 1.6.9 EPS-677

### PR DESCRIPTION
**Pull request by akshayurankar48**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -314,7 +314,8 @@ public function submit_request() {
 			$headers  = 'MIME-Version: 1.0' . "\r\n";
 			$headers .= 'Content-type: text/html; charset=iso-8859-1' . "\r\n";
 			$headers .= 'From:' . $name . '<' . $from . '>' . "\r\n";
-			echo mail( $to, $subject, $html, $headers ) ? esc_html_e( 'Thank you!', 'all-in-one-schemaorg-rich-snippets' ) : esc_html_e( 'Something went wrong!', 'all-in-one-schemaorg-rich-snippets' );
+			$result  .= wp_mail( $to, $subject, wp_kses_post( $html ), $headers );
```
Comment (sushmak02) [inline index.php:317]: Remove . from this line

it should be $result = when defining for first time

File: js/cmb.js [hunk 2]
```diff
@@ -68,14 +68,14 @@
...
-            $('.bsf_upload_button').live('click', function () {
+            $('.bsf_upload_button').on('click', function () {
                 var buttonLabel;
                 formfield = $(this).prev('input').attr('name');
                 buttonLabel = 'Use as ' + $('label[for=' + formfield + ']').text();
                 tb_show('', 'media-upload.php?post_id=' + $('#post_ID').val() + '&type=file&bsf_force_send=true&bsf_send_label=' + buttonLabel + '&TB_iframe=true&bsf_file_upload_nonce=' + bsf_ajax_data.ajax_nonce );
                 return false;
             });
-            $('.bsf_remove_file_button').live('click', function () {
+            $('.bsf_remove_file_button').on('click', function () {
```
Comment (sushmak02) [inline js/cmb.js:78]: make sure these actions are tested

File: js/toggle.js [hunk 3]
```diff
@@ -6,12 +6,13 @@ jQuery(document).ready(function() {
...
 		expand_default(selected);
 	jQuery(window).on('load',function () {
-		if(item_type == "none")
+		if(item_type == "none") {
 			item_hidden();
-		else if(selected != "1" && item_type != "none")
-			item_hidden();
-		else
+		} else if(selected != "1" && item_type != "none") {
+			// Remove duplicate call to item_hidden() here
```
Comment (sushmak02) [inline js/toggle.js:12]: nothing is written in this condition

we can combine these 2 conditions - 

if(item_type == "none") {
			item_hidden();
		} else if(selected != "1" && item_type != "none") {
			item_hidden();
		}

**Review by sushmak02 — CHANGES_REQUESTED** [review]
Check PR comments

File: js/toggle.js (hunk 3 above)
Comment (akshayurankar48) [reply js/toggle.js]: done @sushmak02

File: index.php (hunk 1 above)
Comment (akshayurankar48) [reply index.php]: updated @sushmak02

File: js/cmb.js (hunk 2 above)
Comment (akshayurankar48) [reply js/cmb.js:78]: yes

File: js/toggle.js [hunk 4]
```diff
@@ -6,12 +6,13 @@ jQuery(document).ready(function() {
 	else
 		expand_default(selected);
 	jQuery(window).on('load',function () {
-		if(item_type == "none")
+		if(item_type == "none") {
 			item_hidden();
-		else if(selected != "1" && item_type != "none")
+		} else if(selected != "1" && item_type != "none") {
```
Comment (sushmak02) [inline js/toggle.js:11]: if( ( item_type == "none" ) || ( selected != "1" && item_type != "none" ) {
			item_hidden();

can we combine these 2 conditions?

Comment (akshayurankar48) [reply js/toggle.js]: This is updated @sushmak02

**Review by sushmak02 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -80,6 +80,9 @@ Review, Event, People, Product, Recipe, Software Application, Video, Articles et
 
 ## Changelog ##
 
+### 1.6.8.1 ###
+- Improvement: Fixed security audit issues.
```
Comment (sushmak02) [inline README.md:84]: Improved plugin codebase for better security

**Review by sushmak02 — COMMENTED** [review]
Update plugin version to 1.6.9 and needful release related changes

**Review by sushmak02 — APPROVED** [review]

**Review by vijetaR — APPROVED** [review]

**Pull request merged**